services/housetables/src/main/java/com/linkedin/openhouse/housetables/services/UserTablesServiceImpl.java

```diff
@@ -195,7 +232,7 @@ private List<UserTableDto> listTables(UserTable userTable) {
         () ->
             StreamSupport.stream(
                     htsJdbcRepository
-                        .findAllByDatabaseIdIgnoreCase(userTable.getDatabaseId())
+                        .findAllByDatabaseIdIgnoreCaseAndDeletedIsFalse(userTable.getDatabaseId())
```

Comment:
Is the intention here to list soft deleted tables as well?

Reply:
I think list tables should also exclude the deleted tables.

Reply:
This function does filter out soft deleted tables, softDeleted tables are indicated with `deleted=True`, I will update the PR description